pymc3/tests/test_dist_math.py

```diff
@@ -16,7 +16,7 @@
 import numpy as np
 import numpy.testing as npt
 import pytest
-import scipy.special as st
+import scipy.special as sp
```

Comment:
There's just one occurrence of this module in this file.
IMHO it would be more readable to just `import scipy.special` without any aliasing.